requirements.txt

```diff
@@ -4,7 +4,7 @@ requests~=2.26.0
 kubernetes~=18.20.0
 retry~=0.9.2
 waiting~=1.4.1
-assisted-service-client>=1.0.27.1,<=1.0.27.6
+assisted-service-client~=1.0.27.1
```

Comment:
don't like having a const (1.0.27.*) version. why not taking the latest version?
My question is relevant for the old code too

Reply:
This is not changing what we currently have, it's just more idiomatic.
In any case we did this change in the past because later packages are breaking the current code. We will need to upgrade the client soon, but it's not so straightforward - IMO it's more important to stabilize the application first.